src/app/components/mnoe-users-local-list/mnoe-users-local-list.html

```diff
@@ -40,8 +40,13 @@
             <span ng-show="user.status === 'invited'" ng-click="removeUserFromOrganization(user)" uib-tooltip="{{'mnoe_admin_panel.dashboard.users.widget.local_list.cancel_invite' | translate}}"><i class="fa fa-close fa-1"></i></span>
           </td>
           <td ng-if="organizationContext">
-            <select ng-model="user.role" ng-options="role as keyFromRole(role) | translate for role in availableRoles" ng-change="updateUserRole(user, '{{user.role}}')"></select>
+            <select ng-show="user.editMode" ng-model="user.role" ng-options="role as usersRole.keyFromRole(role) | translate for role in usersRole.availableRoles"></select>
```

Comment:
With the new service you've implemented I think we can simplify this by mapping the translations in the controller and just passing the already mapped translations in as options

Reply:
As disussed IRL -- this won't work because the ng-options translations are needed to bind to the ng-model.

Reply:
Could you elaborate on this?

Reply:
In order for `user.role` to be correct it must be translated back from the UI text into the text that is expected on the backend. Having this code in the ng-options does this natively with angular. When you feed ng-options just an array of translated roles, user.roles will be the ui text instead of the backend text.

Reply:
Ah yeah I see, easier to mentally parse this way:
```CoffeeScript
role as (usersRole.keyFromRole(role) | translate) for role in usersRole.availableRoles
```
Makes sense :+1:

Actually, it's possible to do it:

Your service returns an array like this by iterating/translating the `USER_ROLES` constant:
```javascript
[
    {value: 'admin', translatedLabel: 'Admin'},
    {value: 'staff', translatedLabel: 'Staff'}
]
```

Your `ng-options` would then be:
```CoffeeScript
role.value as role.translatedLabel for role in userRoles.availableRolesForOptions
```
(_names to be changed_)

Would that work?
Let me know if you want to implement this way or keep it like this.

Reply:
@ouranos Ah yes I think that would work, but I think it would be better to just leave it here. 

Because in order to get 
`[
    {value: 'admin', translatedLabel: 'Admin'},
    {value: 'staff', translatedLabel: 'Staff'}
]` in the controller you would have to use $q.all and would get a bit messy. Also the change doesn't make it all that much clearer. 

Agreed about putting the parentheses there though.